UI/win-update/updater/CMakeLists.txt

```diff
@@ -1,44 +1,42 @@
-project(updater)
+cmake_minimum_required(VERSION 3.24...3.25)
+
+legacy_check()
 
 option(ENABLE_UPDATER "Build with Windows updater" ON)
 
 if(NOT ENABLE_UPDATER)
-  message(STATUS "OBS:  DISABLED   Windows updater")
+  target_disable_feature(obs "Windows updater")
   return()
 endif()
 
-find_package(zstd)
+find_packge(zstd)
+
+if(NOT TARGET OBS::json11)
+  add_subdirectory("${CMAKE_SOURCE_DIR}/deps/json11" "${CMAKE_BINARY_DIR}/deps/json11")
+endif()
 
 add_executable(updater WIN32)
 
 target_sources(
   updater
-  PRIVATE updater.cpp
-          updater.hpp
-          patch.cpp
+  PRIVATE hash.cpp
+          helpers.cpp
+          helpers.hpp
           http.cpp
-          hash.cpp
+          init-hook-files.c
+          patch.cpp
           resource.h
+          updater.cpp
+          updater.hpp
           updater.rc
-          init-hook-files.c
-          updater.manifest
-          helpers.cpp
-          helpers.hpp
-          ${CMAKE_SOURCE_DIR}/deps/json11/json11.hpp
-          ${CMAKE_SOURCE_DIR}/deps/json11/json11.cpp)
-
-target_include_directories(updater PRIVATE ${CMAKE_SOURCE_DIR}/libobs
-                                           ${CMAKE_SOURCE_DIR}/deps/json11)
+          updater.manifest)
```

Comment:
`updater.manifest` alphabetically comes before `updater.rc`, but unless there's a logical reason to this order.